notification/docs/HowToRun.md

```diff
@@ -64,6 +62,8 @@ Refer to our [docker hub](https://hub.docker.com/r/commercetools/commercetools-a
     commercetools/commercetools-adyen-integration-notification
 ```
 
-### AWS Lambda
+## Deployment
 
-For deployment to AWS Lambda, zip the `notification` folder and specify `src/lambda.js` as the entry point for the AWS Lambda function.
+Notification module supports different deployment [options](/deployment-examples).
```

Comment:
same here could you link to GCP readme ?

Reply:
but that will express the idea that we only support GCP deployment which is not true.

Reply:
1ccf903e31fc2bb38b11f2dca34fc88d09da8879